pkg/kv/kvserver/kvflowcontrol/replica_rac2/processor.go

```diff
@@ -570,7 +511,9 @@ func (p *processorImpl) OnDestroyRaftMuLocked(ctx context.Context) {
 
 // SetEnabledWhenLeaderRaftMuLocked implements Processor.
 func (p *processorImpl) SetEnabledWhenLeaderRaftMuLocked(
-	ctx context.Context, level kvflowcontrol.V2EnabledWhenLeaderLevel,
+	ctx context.Context,
+	level kvflowcontrol.V2EnabledWhenLeaderLevel,
+	raftBasicState RaftNodeBasicState,
```

Comment:
Ditto re dup-less naming: `state`? Like in `processorImpl.HandleRaftReadyRaftMuLocked` below btw.